src/analytics/tracking.js

```diff
@@ -83,12 +82,19 @@ export const trackPageView = (track, store, previousPage, currentPage, getPayloa
   // Don't let sessions expire whilst still browsing
   touchSessionId(store, state);
 
+  const url = getUrlFromPage(currentPage, global);
```

Comment:
are we sure that `currentPage` is always valid?

Reply:
Yes, you can't navigate anywhere without having a page. `previousPage` can be null though